base/boot.jl

```diff
@@ -510,9 +510,11 @@ end)
 
 function Symbol(s::String)
     @_foldable_meta
+    @noinline
```

Comment:
why is this `@noinline`?

Reply:
There's rarely a good reason to inline a ccall and in this case if you inline it, you lose the ability to consteval it, because the effect flags only attach to the function as a whole.